src/Laravel/Cashier/WebhookController.php

```diff
@@ -50,6 +50,16 @@ protected function eventExistsOnStripe($id)
     }
 
     /**
+     * Check if testing mode is on
+     * 
+     * @return bool
+     */
+    protected function isInTestingMode()
+    {
+        return (bool) getenv('CASHIER_TESTING');
```

Comment:
I have thought about this, but when testing you might not always put the APP_ENV to testing.

But in case you have multiple testing frameworks with different APP_ENV then you won't be in test mode.

Adding a CASHIER_TESTING variable allows to put in in testing without having to care about the environment you are using.